.github/actions/run-python-test-set/action.yml

```diff
@@ -123,8 +123,8 @@ runs:
           exit 1
         fi
         if [[ "${{ inputs.run_in_parallel }}" == "true" ]]; then
-          # -n4 uses four processes to run tests via pytest-xdist
-          EXTRA_PARAMS="-n4 $EXTRA_PARAMS"
+          # -n6 uses four processes to run tests via pytest-xdist
```

Comment:
```suggestion
          # -n6 uses six processes to run tests via pytest-xdist
```

Reply:
Thank you, still experimenting with the values and gathering numbers.
I will also try running 20 and see the time difference, maybe the bottlenecks will be more visible.

Reply:
So far:
* 20 was not allowed as too big value
* 16 always has some flacky tests
* 8 and 6 have a few seconds increase, I will run them more to see which is less flacky